src/component/structure-component.ts

```diff
@@ -255,7 +255,7 @@ class StructureComponent extends Component {
 
   addRepresentation <K extends keyof StructureRepresentationParametersMap>(
     type: K,
-    params: Partial<StructureRepresentationParametersMap[K]> = {},
+    params: Partial<StructureRepresentationParametersMap[K]>|{defaultAssembly: any} = {},
```

Comment:
This should be added to structure-representation.ts, l:38 and typed as a `string` (not `any`)

Reply:
Hmm, I can't make that work. I tried adding it there, but I still get a tsc err, because (as I understand it, admittedly imperfectly), `defaultAssembly` would be needed in all elements of `StructureRepresentationParametersMap` (i.e. all mapped values of `K`), not just `StructureRepresentationParameters`.
```
% tsc
src/component/structure-component.ts:261:12 - error TS2339: Property 'defaultAssembly' does not exist on type 'Partial<StructureRepresentationParametersMap[K]>'.

261     params.defaultAssembly = this.parameters.defaultAssembly
               ~~~~~~~~~~~~~~~
```

Reply:
But this does work (with adding `defaultAssembly` to `StructureRepresentationParameters`):
```typescript
addRepresentation <K extends keyof StructureRepresentationParametersMap>(
    type: K,
    params: Partial<StructureRepresentationParametersMap[K]>|Partial<StructureRepresentationParameters> = {},
    hidden = false
  ) {
    params.defaultAssembly = this.parameters.defaultAssembly
```
Would that be OK?

Reply:
Well, I've checked and it appears that some representations such as label extend their parameters from `RepresentationParameters` and not `StructureRepresentationParameters` which might be the cause.
After better reading at the code it appears that the `defaultAssembly` parameter is not exposed to the user (ie, the user can not set it via `addRepresentation`), so it does not really make sense to add it to the list of parameters in `StructureRepresentationParameters`. 
Your initial typings now seems to me like the best thing to do. Sorry for the noise...

Reply:
No problem -- always good to try for the best solution.